modules/migrations/gitea.go

```diff
@@ -217,12 +218,12 @@ func (g *GiteaLocalUploader) CreateReleases(releases ...*base.Release) error {
 			}
 			defer resp.Body.Close()
 
-			localPath := attach.LocalPath()
-			if err = os.MkdirAll(path.Dir(localPath), os.ModePerm); err != nil {
-				return fmt.Errorf("MkdirAll: %v", err)
+			fs := storage.FileStorage{
```

Comment:
This is trickier. Depeding on the current version value you should or should not ignore the app.ini setting and use local file storage in this function.